pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -191,3 +191,63 @@ def test_dataframe_constructor_with_different_dtype_raises():
     xpr = "Cannot coerce extension array to dtype 'int64'. "
     with tm.assert_raises_regex(ValueError, xpr):
         pd.DataFrame({"A": arr}, dtype='int64')
+
+
+class TestArithmeticOps(BaseDecimal, base.BaseArithmeticOpsTests):
+
+    def check_opname(self, s, op_name, other, exc=None):
+        super(TestArithmeticOps, self).check_opname(s, op_name,
+                                                    other, exc=None)
```

Comment:
Is this needed? It seems you are not changing anything in the implementation

Reply:
Yes, this is needed because the super `check_opname` has a default of `exc=NotImplementedError`.  So this overrides that default.